types/prosemirror-model/index.d.ts

```diff
@@ -1241,6 +1242,11 @@ export interface NodeSpec {
      * to a string representation for debugging (e.g. in error messages).
      */
     toDebugString?: ((node: ProsemirrorNode) => string) | null | undefined;
+    /**
+     * @note Require `prosemirror-dropcursor` module.
+     * This property controls the showing of a drop cursor inside the node.
+     */
+    disableDropCursor?: boolean | ((view: EditorView, pos: number) => boolean);
```

Comment:
This prop is mentioned in the PM changelog, but not in the PM docs. Please open a PR with PM that adds this prop to the docs, too. Also please add the version from which disableDropCursor is supported in prosemirror-dropcursor. (1.4.0)